src/battle-log.ts

```diff
@@ -655,7 +655,7 @@ class BattleLog {
 				return false;
 			},
 			getURI(uri: string) {
-				return 'http://pokemonshowdown.com/interstice?uri=' + encodeURIComponent(uri);
+				return `http://${window.Config?.routes.root || 'pokemonshowdown.com'}/interstice?uri=${encodeURIComponent(uri)}`;
```

Comment:
```suggestion
				return `http://${Config.routes.root}/interstice?uri=${encodeURIComponent(uri)}`;
```

This file newly depends on config so we don't need stuff like this anymore.